packages/babel-plugin-check-es2015-constants/test/fixtures/general/no-assignment/expected.js

```diff
@@ -0,0 +1,4 @@
+var MULTIPLIER = 5;
+MULTIPLIER = (function () {
```

Comment:
I'm sure these would pass, but would be great to add test for destructuring assignments, e.g.

```js
const a = 1;
[a] = [2];

const b = 1;
({b} = {b: 2})
```